src/sbt-test/sbt-guardrail/scala-client-codegen-app/src/main/scala/Hello.scala

```diff
@@ -0,0 +1,7 @@
+package minproject
+
+object Hello {
+  def main(args: Array[String]) = {
+    println("Hello World!!")
```

Comment:
`Client.httpClient(Route.asyncHandler(Resources.routes(new Handler { ... }))).foo()` would be helpful to ensure the generated files are actually exercised.